st2client/st2client/config_parser.py

```diff
@@ -79,6 +79,10 @@
         'password': {
             'type': 'string',
             'default': None
+        },
+        'apikey': {
```

Comment:
The whole story started with https://github.com/StackStorm/st2-packages/issues/283 which implies there is a valid usecase to have api key in config.